frontend/public/components/routes.tsx

```diff
@@ -267,6 +222,25 @@ const RouteTargetRow = ({route, target}) => <tr>
   <td>{calcTrafficPercentage(target.weight, route)}</td>
 </tr>;
 
+const IngressDetails: React.SFC<IngressDetailsProps> = ({ingresses}) => <React.Fragment>
+  {_.map(ingresses, (ingress, i) => <React.Fragment key={i}>
+    <h1 className="co-section-title">{ingress.host}</h1>
+    <table className="co-m-route-horizontal-props">
```

Comment:
Use a `dl` instead of a `table`

Reply:
Hi, a little confused :-) The default `dl, dt, dd` defined in `_common.scss & _base.scss` make the `dt` uppercase and ontop of the `dd`, like we see in the Detail pages.  I thought we wanted: 'label: value', side-by-side?  I've only seen one example of this, but it used `<spans>'s` and only matched up due to the labels being the same number of chars.

Reply:
I think that the existing dl, dt, dd styles are fine.